tests/libnet/cluster.go

```diff
@@ -0,0 +1,38 @@
+package libnet
+
+import (
+	"fmt"
+
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	netutils "k8s.io/utils/net"
+
+	"kubevirt.io/client-go/kubecli"
+)
+
+func IsClusterDualStack(virtClient kubecli.KubevirtClient) (bool, error) {
+	// grab us some neat kubevirt pod; let's say virt-handler is our target.
+	targetPodType := "virt-handler"
+	virtHandlerPod, err := getPodByKubeVirtRole(virtClient, targetPodType)
+	if err != nil {
+		return false, err
+	}
+	for _, ip := range virtHandlerPod.Status.PodIPs {
+		if netutils.IsIPv6String(ip.IP) {
+			return true, nil
+		}
+	}
+	return false, nil
+}
+
+func getPodByKubeVirtRole(virtClient kubecli.KubevirtClient, kubevirtPodRole string) (*k8sv1.Pod, error) {
+	labelSelectorValue := fmt.Sprintf("%s = %s", "kubevirt.io", kubevirtPodRole)
```

Comment:
Please -
1. Substitute "kubevirt.io" `kubevirt.io/client-go/api/v1 - AppLabel`.
2. Substitute `k8sv1.NamespaceAll` with `flags.KubeVirtInstallNamespace`.

Reply:
Done; thanks for looking this up. I should have done a simple 'kubevirt.io' search to find the constant.